### ISSUE TITLE
[question] Converting ontology format

### ISSUE THREAD

**Issue by az-ihsan** [hsolbrig/funowl#57]
Hi, 

Is there any chance that I can convert the ontology written in functional syntax into rdf based format with funowl?

THank you.

**Comment by hsolbrig** [comment]
Yes -- that is one thing it is pretty good at, although it takes a bit 'o time if you are loading something the size of SNOMED-CT.   I'll add some instructions on how to use the CLI a little later today.

**Comment by az-ihsan** [comment]
Perfect, looking forward!

**Comment by hsolbrig** [comment]
So it turns out that I never created an official CLI.  Will correct this shortly, but a code example can now be found in the README of https://github.com/Harold-Solbrig/funowl/tree/issue_57 as well as an example in the tests/test_issues/test_issue_57 directory.  Note that there are a couple of useful options in the `to_rdf` function...

**Comment by az-ihsan** [comment]
Thank you very much! it works perfectly now to my problem..

**Comment by hsolbrig** [comment]
Pull request should be good to go.  Tried to add you as a reviewer, but apparently need to go through github hoops.  Will deploy it shortly.
